docs/keto/sdk/go.md

```diff
@@ -0,0 +1,231 @@
+---
+id: go
+title: Go
+---
+
+import CodeBlock from '@theme/CodeBlock'
+import { useLatestRelease } from '@site/src/hooks'
+
+
+Ory Keto exposes two APIs for integration
+
+- [gRPC](http://ory.sh/docs/keto/reference/proto-api)
+- [REST](http://ory.sh/docs/keto/reference/rest-api)
+
+## Installation
+
+### Installation gRPC API
+
+```mdx-code-block
+
+<CodeBlock className="language-shell">{`go get github.com/ory/keto/proto/ory/keto/acl/v1alpha1@${useLatestRelease('keto')}`}</CodeBlock>
+
+```
+
+### Installation REST API
+
+```mdx-code-block
+<CodeBlock className="language-shell">{`go get github.com/ory/keto-client-go${useLatestRelease('keto')}`}</CodeBlock>
+```
```

Comment:
I think useLatestRelease causes the command to be 

`go get github.com/ory/keto-client-gov0.8.0-alpha.2`

but we need 
`go get github.com/ory/keto-client-go@v0.8.0-alpha.2`